### PR TITLE
PORTALS-1851: use portal header logo (instead of footer logo)

### PR DESCRIPTION
**Pull request by jay-hodgson**
and callback banner background will be white

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Screen Shot 2021-02-22 at 4 35 12 PM

**Comment by jay-hodgson** [comment]
Related to https://github.com/Sage-Bionetworks/SynapseWebClient/pull/4520

File: src/AppInitializer.tsx [hunk 1]
```diff
@@ -200,22 +200,19 @@ class AppInitializer extends React.Component<Props, AppInitializerState> {
...
     let icon = ''
     const footerElement = document.querySelector('#footer')
     if (footerElement) {
       color = window
-        .getComputedStyle(footerElement, null)
-        .getPropertyValue('color')
-      background = window
         .getComputedStyle(footerElement, null)
         .getPropertyValue('background-color')
     }
```
Comment (emmanmills) [inline src/AppInitializer.tsx:212]: You delete the background statement and not the color state in `if (footerElement)` block. Is this intentional?

**Review by emmanmills — COMMENTED** [review]
Just a comment to make sure I am not missing anything. Otherwise looks good.

File: src/AppInitializer.tsx (hunk 1 above)
Comment (jay-hodgson) [reply src/AppInitializer.tsx:212]: Yep, it now uses the footer background color as the themed color in the back-bar.  And the back-bar background color is just white now.

**Review by emmanmills — APPROVED** [review]

**Pull request merged**